shared/common/ui/select/index.tsx

```diff
@@ -201,6 +233,7 @@ export function Select<T extends any>({
                         [styles.dropdownItemSelected]:
                           dropdown.selectedItemId === result.obj.item.item.id,
                         [styles.disabled]: !!result.obj.item.item.disabled,
+                        [styles.fullScreen]: !!fullScreen,
```

Comment:
Rather than setting this class on every item, I would just have the one on the dropdown element.

Reply:
I will move font styles upper, the padding and border is needed per item though.